torch/distributed/device_mesh.py

```diff
@@ -373,8 +378,11 @@ def _setup_world_group_and_device(self):
 
             return _get_default_group()
 
+        @staticmethod
         def _init_process_groups(
-            self,
+            layout: _MeshLayout,
+            global_rank_permutation: torch.Tensor,
+            mesh_dim_names: Optional[tuple[str, ...]],
             backend_override: tuple[BackendConfig, ...],
         ):
```

Comment:
Looks like we are changing the return signature of this function, can you also update it here?